tests/hazmat/primitives/utils.py

```diff
@@ -422,24 +423,9 @@ def kbkdf_counter_mode_test(backend, params):
         "hmac_sha512": hashes.SHA512,
     }
 
-    supported_counter_locations = {
-        "before_fixed": CounterLocation.BeforeFixed,
-        "after_fixed": CounterLocation.AfterFixed,
-    }
-
-    algorithm = supported_algorithms.get(params.get("prf"))
-    if algorithm is None or not backend.hmac_supported(algorithm()):
-        pytest.skip(
-            "KBKDF does not support algorithm: {}".format(params.get("prf"))
```

Comment:
I don't entirely follow why this is no longer a relevant skip. What captures this now?

Reply:
Test vectors contain only KBKDFHMAC and KBKDFCMAC test data. Before this PR, the latter triggered this condition. Now, that all algorithms are covered, "prf" checks have to be replaced by asserts (lines 427, 428, 459, 500). Otherwise, codecov would complain.